### DISCUSSION TITLE
What would be the best way to import a 2D DXF file in MakerJS

### DISCUSSION THREAD

**Discussion by maelp** [microsoft/maker.js#573] [General]
I need to convert a DXF file containing elements to a polygon, using either the "outlining" function of MakerJS or the "toPoints" method to sample points regularly,

what would be the best way to do this?

Thank you

**Comment by danmarshall** [comment]
Hello, MakerJs doesn't have DXF import. There are a few on npm: https://www.npmjs.com/search?q=dxf
